hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -23,24 +24,40 @@
 
   // Import additional common sim cfg files.
   // TODO: remove imported cfgs that do not apply.
-  import_cfgs: [// Project wide common sim cfg file
-                "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+  import_cfgs: [
+      // Project wide common sim cfg file
+      "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+      // Config file to pull in the correct flags for otbn_memutil
+      "{proj_root}/hw/ip/otbn/dv/memutil/otbn_memutil_sim_opts.hjson"
```

Comment:
should we import some common CSR test, mem test?

Reply:
Yep, definitely. I'll do that in a later PR, when I actually get that stuff working.